lib/RTCSession.js

```diff
@@ -1046,13 +1047,32 @@ module.exports = class RTCSession extends EventEmitter
       }
       else
       {
-        const dtmf = new RTCSession_DTMF(this);
+        // send DTMF according to transport config
+        switch (transportType)
```

Comment:
Comments always with proper capital letter and dot at the end.

```suggestion

        // Send DTMF according to transport config.
        switch (transportType)
```